tjdashboard/code/administrator/tjdashboard.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="component" version="3.0" method="upgrade">
+    <name>com_tjdashboard</name>
+    <creationDate>2017-09-15</creationDate>
+    <copyright>2017 Techjoomla</copyright>
+    <license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+    <author>Techjoomla</author>
+    <authorEmail>extensions@techjoomla.com</authorEmail>
+    <authorUrl>http://techjoomla.com</authorUrl>
+    <version>CVS: 1.0.0</version>
+    <description></description>
+
+    <install> <!-- Runs on install -->
```

Comment:
Where is the updates section ?

Reply:
@coolbung Little bit confused , Update about sql(tables)  or updateserver ?
1. To update the tables structure
     <update> <!-- Runs on Update -->
         <schemas>
	       <schemapath type="mysql">sql/updates/mysql</schemapath>
	</schemas>
     </update>
2. To upgrade the 
<updateservers>
    <server type="extension" priority="1" name="Extension Update Site">http://example.com/extension.xml</server>
    </updateservers>

Reply:
OK